en/docs/insights/derive-insights.md

```diff
@@ -0,0 +1,364 @@
+ 
+    ![API usage by backend](../assets/img/insights/usage-by-backend.png){.cInlineImage-half}
+ 
+ - **API Resource Usage**
+ 
+    This table shows a resource-level breakdown of API traffic. Each row represents an API resource and it shows the API name, resource path, API method, and the hit count for that combination.
+    
+    ![API resource usage](../assets/img/insights/resource-usage.png){.cInlineImage-half}
+    
+### Errors
+
+This page shows information related to erroneous API calls that are received by your system. The errors are categorization based on the error type. You can further drill down using the error subtypes. Use this page as the starting point for debugging any API errors.
+
+![Error statistics](../assets/img/insights/error-page-full.png){.cInlineImage-full}
+
+You can filter the information displayed in the widgets as follows:
+
+| **Filtering Option** | **Description** |
+|--------------------|-------------------------------------------------------------------------------------------------|
+| **By API** | In the **API** field, select the APIs for which you want to view analytics. By default, all APIs are selected. You can select one or multiple APIs from this selector and view the aggregated result. |
+| **By Category** | The **Category** field is displayed when you click **Category** in the toggle switch in the upper right corner of the page. In this field, you can select one or all of the following error categories:<br/>**Authentication**: Any kind of authentication error falls into this category including expired/missing/invalid credentials.<br/>**Target Connectivity**: Any kind of API back-end error falls into this category including connection time-outs/other back-end errors (for example, 4xx and 5xx status codes).<br/>**Throttling**: Any kind of API back-end error falls into this category including application throttling/subscription throttling.<br/>**Other**: All other errors fall into this category including mediation errors/resource not found errors. |
+| **By Status Code** | The **Status Code** field is displayed when you click **Status Code** in the toggle switch in the upper right corner of the page. In this field, you can select all or any available HTTP status code category (4xx, 5xx, etc.). These status codes represent the response status of the API Gateway. |
+
+#### Category mode
+
+The following widgets are available for monitoring errors when you have selected the category mode via the toggle switch in the upper-right corner of the page.
+
+- **Errors by Category** (graph)
+
+    This graph shows the error category distribution over time for the selected period. Apply the required filters as explained above to select the APIs and the error categories to which this content applies. When you select multiple APIs, the error count is grouped by category.
```

Comment:
error category distribution -> errors by category over time for the selected period.

Reply:
Apply the required filters as explained above to select the APIs and the error categories to customize the result.

Reply:
Done